RecoDataProducts/src/ComboHit.cc

```diff
@@ -221,6 +221,19 @@ namespace mu2e {
     return retval;
   }
 
+  float ComboHitCollection::eDepAvg(ComboHitCollection const& ccol) {
```

Comment:
I don't understand why a ComboHitCollection is provided as argument since this is a member function of ComboHitCollection. This function can just act on its own object, with no argument. Also the function should be declared 'const'.